packages/atlas/src/components/_nft/NftCard/NftCard.styles.ts

```diff
@@ -7,7 +7,137 @@ import { cVar, media, sizes } from '@/styles'
 
 export const Container = styled.div<{ fullWidth?: boolean }>`
   width: ${({ fullWidth }) => (fullWidth ? '100%' : '360px')};
-  background-color: ${cVar('colorBackgroundStrong')};
+  transform-style: preserve-3d;
+  perspective: 2000px;
+  position: relative;
+`
+
+export const FadeMask = styled.div`
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  mask-position: 50% 50%;
+  transition: all 0.1s ease;
+  mask-image: linear-gradient(
+    104deg,
+    rgb(0 0 0 / 1) 0%,
+    rgb(0 0 0 / 1) 7%,
+    rgb(0 0 0 / 0.5) 10%,
+    rgb(0 0 0 / 0) 40%,
+    rgb(0 0 0 / 0) 60%,
+    rgb(0 0 0 / 0.5) 90%,
+    rgb(0 0 0 / 1) 92%,
+    rgb(0 0 0 / 1) 100%
+  );
+  mask-mode: alpha;
+  mix-blend-mode: hard-light;
+  opacity: 0;
+`
+
+export const Pattern = styled.div`
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  background-image: url('https://i.imgur.com/AjBArhU.png');
```

Comment:
I think we shouldn't take it from external server, it should live in our assets directory.